notebooks/7-rotation-speeds/interm_rotation_rate.ipynb

```diff
@@ -0,0 +1,2409 @@
+{
```

Comment:
<p>Since this is an intermediate notebook, could we have a brief exercise here where the reader checks the quality flags for those peaks? (Like maybe, check the quality flags for any points in the time series with flux greater than 1900 e/s) And we could refer them to the earlier lesson Emma's working on with the quality flags, or other resources.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/spacetelescope/project-tikebook/pull/12/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='9'/>